### PR TITLE
Some linting

### PR DESCRIPTION
**Pull request by phillipm**

### CODE REVIEW

File: javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java [hunk 1]
```diff
@@ -34,26 +30,14 @@ public int getAdditional() {
     }
 
     public void addBadImageReference() {
-        badImageReferenceCount++;
+        // TODO: remove this
```
Comment (amstone326) [inline javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java]: why are you not just removing this?

Comment (phillipm) [inline javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java]: It is sorta hard to fully remove it because it is used by the core versions of the suite/xform installer's `verifyInstallation`. That code isn't run currently on the CLI, but we might want to enable it. Figuring out the right path forward seems like way more work than I want to do for this right now. I can revert the changes in this file if you'd like.

Comment (amstone326) [inline javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java]: Got it, why are you removing the method body then? I don't understand the purpose of leaving the method in, but leaving it empty.

Comment (phillipm) [inline javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java]: `badImageReferenceCount` is never read, so keeping it around didn't seem useful

Comment (amstone326) [inline javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java]: Ok, I think it probably makes more sense to revert the changes to this file for now than leaving it in a half-but-not-fully-deleted state
